mathesar_ui/src/App.svelte

```diff
@@ -19,16 +20,26 @@
     <Button on:click={() => newImport(selectedDb)}>
       Import CSV
     </Button>
+    <a href="schemas/">
+      Manage schemas
+    </a>
   </div>
 </header>
 
 <section class="content-section">
-  <Route path="/:db">
-    {#key selectedDb}
-      <Base database={selectedDb}/>
-    {/key}
+  <Route path="/:db/*">
+    <Route path="/schemas">
+      {#key `${selectedDb}__schemas`}
```

Comment:
The key tag is used to rerender its children when the value changes. It would suffice to only mention `selectedDb` here.

Reply:
Updated in 23c5be5.